README.md

```diff
@@ -31,15 +28,22 @@ To get a local copy, run the fllowing command.
 
 ### Usage
 
-`npm start`
+- `npm run build`
+- `npm start`
 
 ## Authors
 
 👤 **Author**
 
 - GitHub: [@KanzaTahreem](https://github.com/KanzaTahreem)
 - Twitter: [@kanza_tahreem](https://twitter.com/kanza_tahreem)
-- LinkedIn: [LinkedIn](https://www.linkedin.com/in/kanza-tahreem/)
+- LinkedIn: [Kanza Tahreem](https://www.linkedin.com/in/kanza-tahreem/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
 
 ## Show your support
 
```

Comment:
- [ **OPTIONAL** ] **RECOMMENDED:** Kindly make sure you add the **[MIT.md](https://github.com/microverseinc/readme-template/blob/master/MIT.md)** file in your root directory and edit it if possible according to your information to make your README more professional. Right now. when the user clicks this link `[MIT](./LICENSE)` it shows an error page.